<?xml version="1.0" encoding="utf-8"?>
<!-- This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com) 
     Commit Version: 1.6.0-4-g7f85cfe  Build Version: 1.6.7995.38578
     For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
<robot
  name="Pierna_entera_2">
  <link
    name="base_link">
    <inertial>
      <origin
        xyz="-0.00250245032232071 0.251517073292659 0.175881539388643"
        rpy="0 0 0" />
      <mass
        value="0.313191197329196" />
      <inertia
        ixx="0.00113074597450617"
        ixy="5.06648436970605E-05"
        ixz="1.01441767230027E-05"
        iyy="0.000112285853478866"
        iyz="-6.28525512292321E-05"
        izz="0.00113161900695232" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://lesson_urdf/meshes/base_link.STL"/>
      </geometry>
      <material
        name="">
        <color
          rgba="0.898039215686275 0.92156862745098 0.929411764705882 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://lesson_urdf/meshes/base_link.STL" />
      </geometry>
    </collision>
  </link>
  <link
    name="link_p">
    <inertial>
      <origin
        xyz="0.00033555 -0.0029601 0.020892"
        rpy="0 0 0" />
      <mass
        value="0.13838" />
      <inertia
        ixx="5.0666E-05"
        ixy="-1.549E-06"
        ixz="1.1666E-07"
        iyy="6.4116E-05"
        iyz="-1.0334E-06"
        izz="4.273E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://lesson_urdf/meshes/link_p.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 0 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://lesson_urdf/meshes/link_p.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint_p"
    type="continuous">
    <origin
      xyz="-0.0024512 -0.097818 0.25643"
      rpy="1.5091 0 -3.1416" />
    <parent
      link="base_link" />
    <child
      link="link_p" />
    <axis
      xyz="0 0 1" />
  </joint>
  <link
    name="link_c">
    <inertial>
      <origin
        xyz="0.29751 0.1045 -0.010047"
        rpy="0 0 0" />
      <mass
        value="0.30893" />
      <inertia
        ixx="0.00014292"
        ixy="-0.00020178"
        ixz="-4.7007E-06"
        iyy="0.00064636"
        iyz="-1.6663E-06"
        izz="0.00073083" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://lesson_urdf/meshes/link_c.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 0 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://lesson_urdf/meshes/link_c.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint_c"
    type="continuous">
    <origin
      xyz="-0.0047758 0.041904 0.02325"
      rpy="1.5708 0 -3.0281" />
    <parent
      link="link_p" />
    <child
      link="link_c" />
    <axis
      xyz="0 0 -1" />
  </joint>
  <link
    name="link_r">
    <inertial>
      <origin
        xyz="0.20735 -0.045612 -0.011448"
        rpy="0 0 0" />
      <mass
        value="0.1843" />
      <inertia
        ixx="6.2303E-05"
        ixy="0.00013228"
        ixz="-1.5195E-10"
        iyy="0.00063454"
        iyz="-2.117E-10"
        izz="0.00066576" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://lesson_urdf/meshes/link_r.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 0 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://lesson_urdf/meshes/link_r.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint_r"
    type="continuous">
    <origin
      xyz="0.44154 0.15513 0.037775"
      rpy="0 0 0" />
    <parent
      link="link_c" />
    <child
      link="link_r" />
    <axis
      xyz="0 0 -1" />
  </joint>
</robot>